backend/src/main/java/com/bakdata/conquery/models/query/statistics/ColumnStatsCollector.java

```diff
@@ -18,49 +17,43 @@ public abstract class ColumnStatsCollector<T> {
 	private final String name;
 	private final String label;
 	private final String description;
-	private final ResultType type;
 	@JsonIgnore
 	private final PrintSettings printSettings;
 
-	public static ColumnStatsCollector getStatsCollector(ResultInfo info, final PrintSettings printSettings, BooleanSupplier samplePicker, ResultType type, UniqueNamer uniqueNamer) {
+	public static ColumnStatsCollector getStatsCollector(ResultInfo info, final PrintSettings printSettings, ResultType type, UniqueNamer uniqueNamer, FrontendConfig config) {
 
 		// List recursion must be done before assigning uniqueNames
-		if (type instanceof ResultType.ListT listT){
-			final ColumnStatsCollector<?> columnStatsCollector = getStatsCollector(info, printSettings, samplePicker, listT.getElementType(), uniqueNamer);
+		if (type instanceof ResultType.ListT listT) {
+			final ColumnStatsCollector<?> columnStatsCollector = getStatsCollector(info, printSettings, listT.getElementType(), uniqueNamer, config);
 			// name label type are discarded when using ListColumnStatsCollector
-			return new ListColumnStatsCollector<>(null, null, null, type, columnStatsCollector, printSettings);
+			return new ListColumnStatsCollector<>(null, null, null, columnStatsCollector, printSettings);
 		}
 
-
 		final String name = uniqueNamer.getUniqueName(info);
 		final String label = info.defaultColumnName(printSettings);
 
-		if (type instanceof ResultType.IntegerT) {
-			return new NumberColumnStatsCollector(name, label, info.getDescription(), type, samplePicker, printSettings);
-		}
+		if (type instanceof ResultType.NumericT || type instanceof ResultType.MoneyT || type instanceof ResultType.IntegerT) {
+			final int lowerPercentile = config.getVisualisationPercentiles().hasLowerBound() ? config.getVisualisationPercentiles().lowerEndpoint() : 0;
+			final int upperPercentile = config.getVisualisationPercentiles().hasUpperBound() ? config.getVisualisationPercentiles().upperEndpoint() : 100;
```

Comment:
Die Limits würde ich in der Config enforcen, z.B. über eine ValidationMethod

Reply:
das ist doch schon über die Range gelöst, aber ich müsste sie natürlich auf 0-100 validieren 👍

Reply:
Hatte ich übersehen, ist umgesetzt.